lib/errorHandlers/suppressError.ts

```diff
@@ -1,7 +1,7 @@
-// @ts-nocheck
 const {
   isSpecifiedError,
   isMissingScopeError,
+  ApiErrorContext,
```

Comment:
I believe this failure is because you are importing the type from LDL and not the Class from `./index`

Reply:
Addressed in https://github.com/HubSpot/hubspot-cli/pull/1251/commits/146dcbafeabd6d865cb305cb61cf867207c31fbf